README.md

```diff
@@ -1 +1,75 @@
-# microverse-leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> A minimalist Leaderboard app
+
+![screenshot](./img/screenshot.png)
+
+## Getting Started
+
+### Prerequisites
+
+- [Webpack](https://webpack.js.org/)
+
+### Setup
+
+* Clone this repository
+```
+git clone https://github.com/crgc/microverse-leaderboard.git
+```
+* Change into folder
+```
+cd microverse-leaderboard
+```
+* Install Webpack
+```
+npm install webpack webpack-cli --save-dev
+```
+* Install dependencies
+```
+npm install --save lodash
+npm install --save-dev style-loader css-loader
+npm install --save-dev html-webpack-plugin
+npm install --save-dev copy-webpack-plugin
+npm install --save-dev webpack-dev-server
+```
```

Comment:
- [ ] **OPTIONAL**
This could be shortened to just `npm install` or `npm i` and it'll install all the necessary dependencies and packages the app needs to run 😄

Reply:
Done.